client/rest-high-level/build.gradle

```diff
@@ -26,9 +26,9 @@ group = 'org.elasticsearch.client'
 dependencies {
   compile "org.elasticsearch:elasticsearch:${version}"
   compile "org.elasticsearch.client:rest:${version}"
-  compile "org.elasticsearch.plugin:parent-join-client:${version}"
-  compile "org.elasticsearch.plugin:aggs-matrix-stats-client:${version}"
```

Comment:
does this mean that we don't distribute the modules by default with the high level client?